lib/src/constants.dart

```diff
@@ -23,17 +23,19 @@ enum ChannelEvents {
   accessToken,
   broadcast,
   presence,
-  postgresChanges;
+  postgresChanges,
+}
 
-  static ChannelEvents fromName(String type) {
+extension ChannelEventsExtended on ChannelEvents {
+  static ChannelEvents fromType(String type) {
     for (ChannelEvents enumVariant in ChannelEvents.values) {
-      if (enumVariant.name == type) return enumVariant;
+      if (enumVariant.name == type || enumVariant.eventName() == type) {
```

Comment:
Is `enumVariant.name` usable on Dart versions not supporting this? Like, doesn't it throw an `MethodNotFoundException` on Dart 2.12?

Reply:
Great call! Let me add a test to check the code against v2.12.0 of Dart!

Reply:
And you're right. `.name` was only introduced in 2.15.0. 
https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#dartcore-4

Will fix by adding a name extension method.

Also, Github Actions with specifying the Dart version doesn't seem to be working
https://github.com/dart-lang/setup-dart/issues/61